_troubleshooting/delete_security_group.md

```diff
@@ -23,10 +23,11 @@ neutron lbaas-loadbalancer-list | grep ACTIVE | awk '{print $2}' |xargs -i
 neutron lbaas-loadbalancer-delete {} 
 ```
```

Comment:
On this whole section above, it's unclear to me if that's supposed to be a single command, or a string of commands? It may be worth showing the output, and starting each command with a command prompt.